content/en/docs/tasks/configure-pod-container/create-hostprocess-container.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Create a Windows HostProcess Container
+content_type: task
+weight: 20
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.22" state="alpha" >}}
+
+Windows HostProcess containers enable you to run containerized administrative workloads on a Windows host. These containers operate as normal processes and have access to the host network namespace, storage, and devices when given the associated user privileges. HostProcess containers can be used to deploy networking policies, storage configurations, device plugins, kube-proxy, and other components to Windows nodes without the need for dedicated proxies or the direct installation of services on the Windows nodes.
```

Comment:
Please wrap these long lines. It would be very hard for localization teams to track changes in the future.

Reply:
"administrative" is a confusing term. Maybe "privileged" is better?

Reply:
"These containers operate as normal processes and ..."

shall we change "and" to "but"?

Look, because you are not wrapping the long lines, providing comments to your PR also becomes difficult.

Reply:
"networking policies" is this a reference to CNI's?  CNI's are a key use case.